docs/API.md

```diff
@@ -981,26 +981,22 @@ An object with the following properties:
 
 #### `"result"`
 
-There are two different results depending on the test creation method:
-
-In the case of a test created with `start_domain_test`:
-
 * `"creation_time"`: A *timestamp*. The time at which the *test* was enqueued.
 * `"id"`: An integer.
 * `"hash_id"`: A *test id*. The *test* in question.
-* `"params"`: A normalized version `"params"` object sent to
+* `"params"`: See below.
   `start_domain_test` when the *test* was started.
 * `"results"`: A list of *test result* objects.
 
+If the test was created by `start_domain_test` then `"params"` is  normalized
```

Comment:
maybe a missing word: `is normalized version` -> `is a normalized version`

Reply:
Fixed.